src/sonic-host-services/scripts/hostcfgd

```diff
@@ -1015,6 +1082,7 @@ class HostConfigDaemon:
         radius_global = self.config_db.get_table('RADIUS')
         radius_server = self.config_db.get_table('RADIUS_SERVER')
         self.aaacfg.load(aaa, tacacs_global, tacacs_server, radius_global, radius_server)
+        self.pamLimitsCfg.load(self.config_db)
```

Comment:
>load [](http://example.com/codeflow?start=26&length=4)

Please be aware there is a comprehensive refactor (with some improvement) PR [https://github.com/Azure/sonic-buildimage/pull/10168](https://github.com/Azure/sonic-buildimage/pull/10168).

This `load` could be move into `__init__()`. All the code in `load` are relating to subscribed tables.

Reply:
Fixed, move the load code to _init_()

Reply:
After move code to _init_, I found hostcfgd continuously write PAM error message to syslog, also 'frontend' process take almost 100% CPU, however the config file rendered correctly, so I will check why this happpen and fix it soon.

Reply:
The UT issue and 'frontend' process issue fixed by remove unnecessary command, the pam-auth-update code request user input which cause high CPU utilization in pamd, also will update 201811 PR.